.github/workflows/build.yml

```diff
@@ -4,6 +4,7 @@ on: [push, pull_request]
 
 jobs:
   jvm:
+    if: ${{ github.repository_owner == 'square' }} # patch to reduce merge conflicts
```

Comment:
I would leave this be, I dont see a reason to not do this.

Reply:
I disabled the workflow for some peculiarities. See https://github.com/liachmodded/javapoet/runs/3704516655 (and the two test I modified because of library and java bumps)